tests/test_core.py

```diff
@@ -92,7 +90,6 @@ def test_stream_logs(self, openshift):
         (flexmock(openshift)
             .should_receive('_get')
              # First: timeout in response after 100s
-            .and_raise(ex)
```

Comment:
Turns out we can: https://github.com/projectatomic/osbs-client/commit/3323f30fb63c1b027f9327e799064ea7aeba69fe, and the test fails.